lib/pages/chat/chat.dart

```diff
@@ -679,18 +679,15 @@ class ChatController extends State<ChatPageWithRoom>
         );
 
         if (msgEventId != null) {
-          pangeaController.myAnalytics.setState(
+          pangeaController.putAnalytics.setState(
             AnalyticsStream(
               eventId: msgEventId,
               roomId: room.id,
-              constructs: [
```

Comment:
Removing grammar construct uses. We're not longer saving these. The info from these is now within the negative points on morphs.

Reply:
Let's talk about this though. Maybe I'm missing something.